.gitignore

```diff
@@ -1,3 +1,4 @@
 build
 composer.lock
 vendor
+.idea
```

Comment:
👎 This should be added to your system level ignore file.

Reply:
Agreed